run_service.sh

```diff
@@ -347,7 +347,8 @@ prompt_use_staking() {
     while true; do
         echo "Use staking?"
         echo "------------"
-        read -p "Do you want to use this service in a staking program? (yes/no): " use_staking
+        echo "The current staking program for Trader service on Gnosis is 'Everest'."
```

Comment:
This will have to be updated all the time. Don't do this. Just print the staking contract address if at all